src/Hpack/Config.hs

```diff
@@ -501,6 +509,21 @@ instance Semigroup Empty where
 instance FromValue Empty where
   fromValue _ = return Empty
 
+data GhcLanguage =
+    Haskell98
+  | Haskell2010
+  | GHC2021
+  deriving (Eq, Show, Generic, Enum, Bounded)
+
+instance FromValue GhcLanguage where
+  fromValue = withText $ \ (T.unpack -> t) -> do
+    maybe err return (lookup t options)
+    where
+      err = fail ("expected one of " ++ formatOrList ghcLanguagesAsString)
+      ghcLanguages = [minBound .. maxBound]
+      ghcLanguagesAsString = map show ghcLanguages
+      options = zip ghcLanguagesAsString ghcLanguages
```

Comment:
👎 copy & paste (if we were to keep it, it needs to get DRYed; but see above)

Reply:
This instance definition now falls away, due to the change in the type.